### PR TITLE
Website redesign

### PR DESCRIPTION
**Pull request by lisicase**
## Summary of Changes
Design changes have been made in accordance with the wireframes and follow-up discussion during the OData team meeting.

### Before & After
**Home:**
image

**Get Started:**
image

**Tools:**
image

**Libraries:**
image

**Blog:**
image

**Ecosystem:**
image

## Remaining Work
Remaining tasks (such as those regarding the UI and data storage) detailed in handoff documentation. Of particular note is hooking up the contribution form, which is marked via a TODO in comments in this PR.

### CODE REVIEW

**Review by xuzhg — APPROVED** [review]

File: _data/ecosys-submissions.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+id,product_category,product_name,product_link,download_link,brief_description
+2,Producer,test,testingg,,teaowjei aiwjeg oiajweoj gtaow eia weijgoja wiegjawe o
```
Comment (marabooy) [inline _data/ecosys-submissions.csv:2]: Is this file required? I see some items may have some userids which may have changed for some users.
```suggestion
2,Producer,test,testing,,teaowjei aiwjeg oiajweoj gtaow eia weijgoja wiegjawe o
```

File: _includes/navbar.html [hunk 2]
```diff
@@ -56,42 +53,22 @@
...
 
-                <form class="navbar-form navbar-left" role="search">
-                    <div class="form-group combobox combobox-list">
+                <form class="navbar-form navbar-right search-bar" role="search" style="margin-left: auto">
+                    <div class="search-bar form-group combobox combobox-list">
                         <input id="main-search-input"
                                type="text"
                                aria-label="Search"
-                               class="form-control st-default-search-input"
+                               class="search-bar-input form-control"
```
Comment (marabooy) [inline _includes/navbar.html:71]: Check to see if the swift type works I doubt it would fail with the css change but probably requires some verification would be required as the js provides accessible search through the dynamic widget.

File: _data/ecosys-submissions.csv (hunk 1 above)
Comment (lisicase) [reply _data/ecosys-submissions.csv:2]: Good catch! This file is used as a sample to demonstrate pulling data from a CSV file to reference when creating a solution for storing/retrieving data, but I should comment out where the file is being used for now.

Comment (lisicase) [reply _data/ecosys-submissions.csv:2]: The "typo" of "testingg" is how the data was submitted to the form, so I'll leave that part as is in case people want to reference the raw form submissions at any point.

File: _includes/navbar.html (hunk 2 above)
Comment (lisicase) [reply _includes/navbar.html:71]: As far as I can tell, the class change doesn't affect the functionality (the scripts appear reference the `main-search-input` ID, and the search results seem the same), but if there's something I'm missing, do let me know.

**Review by marabooy — APPROVED** [review]
